backend/endpoints/news/route.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from ninja import Router
+from django.http.request import HttpRequest
+
+from lib.polygon_api import PolygonAPI, normalise_scores
+
+router = Router()
+
+@dataclass
+class TickerArticleDataclass:
```

Comment:
Turns out I was wrong about the Schema thing - we do need them. Should be something like:

```python
class ArticleSchema(Schema):
	title: str
	...

@router.get(... response=List[ArticleSchema])
def news(..., limit=20) -> List[ArticleSchema]:
	return [...]
```